### PR TITLE
Add list of attribute target constants

### PR DESCRIPTION
**Pull request by Firehed**
I wasn't able to find these anywhere in the Attributes documentation, and needed to use reflection to find the valid values.

I think the constant names are self-explanatory enough to not require additional information.

### CODE REVIEW

File: language/attributes.xml [hunk 1]
```diff
@@ -333,6 +333,18 @@ class MyAttribute
     </para>
    </example>
 
+   <para>The following targets can be specified:</para>
+   
+   <simplelist>
+    <member><literal>Attribute::TARGET_CLASS</literal></member>
```
Comment (cmb69) [inline language/attributes.xml:339]:
```suggestion
    <member><constant>Attribute::TARGET_CLASS</constant></member>
```
Same for the others below.

**Review by cmb69 — CHANGES_REQUESTED** [review]
Thank you for the PR! Besides the comment, this looks good to me.

**Comment by Firehed** [comment]
Done! Thank you for the feedback :)

**Review by cmb69 — APPROVED** [review]
Thank you!

**Pull request merged**